configure.ac

```diff
@@ -3960,7 +3960,7 @@ dnl disable cryptographic authentication
 dnl
 AC_MSG_CHECKING([whether to enable cryptographic authentication methods])
 AC_ARG_ENABLE(crypto-auth,
-AS_HELP_STRING([--enable-crypto-auth],[Enable cryptographic authentication])
+AS_HELP_STRING([--enable-crypto-auth],[Enable cryptographic authentication (default)])
```

Comment:
I think you should drop the "crypto auth" part and go all in on the disabling them all one by one instead.

Reply:
I agree. I'll do that. I'll have a couple of commits addressing this and CMake stuff later this week.

Reply:
Done!